app/geant-exporter/geant-exporter.cc

```diff
@@ -0,0 +1,221 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file geant-exporter.cc
+//---------------------------------------------------------------------------//
+/*! \brief Geant4 particle definition and physics tables exporter
```

Comment:
Since this is a description of the file, you can combine it with the file command:
```c++
/*!
 * \file geant-exporter.cc
 * \brief Geant4 particle definition and physics tables exporter.
 * 
```